tests/settings.py

```diff
@@ -0,0 +1,19 @@
+class Settings(object):
+    """
+    Class that holds settings for each test suite to use, for example which instrument is currently being tested
+
+    Should be used in a static way only.
+
+    These variables are set from the main test runner and read from the test classes when the tests run.
+    """
+
+    name = ""
+    hostname = ""
+    config_repo_path = ""
+    gui_repo_path = ""
+    pv_prefix = ""
+    valid_iocs = None
+    protected_iocs = None
+
+    def __init__(self):
+        raise NotImplementedError("Do not create an instance of this class.")
```

Comment:
`NotImplementedError` is generally reserved in development for something that hasn't been implemented. This has been implemented. As per suggestion above, this shouldn't be necessary.

Reply:
Who is catching this exception?

Reply:
Changed to a `RuntimeError`. I think it is useful to leave it in to make it obvious that instances are never created.